apps/dashboard/src/app/(internal)/event/components/all-attendees-table.tsx

```diff
@@ -44,10 +64,17 @@ export const AllAttendeesTable = ({ attendees, attendance }: AllAttendeesTablePr
   const columnHelper = createColumnHelper<Attendee>()
   const columns = useMemo(
     () => [
-      columnHelper.accessor((attendee) => attendee.user.name, {
+      columnHelper.accessor((attendee) => attendee.user, {
         id: "user",
         header: "Bruker",
-        cell: (info) => info.getValue(),
+        cell: (info) => {
+          const user = info.getValue()
```

Comment:
```suggestion
          const user = info.row.original.user
```
To fix search on name